lib/puppet/functions/simplib/validate_port.rb

```diff
@@ -0,0 +1,39 @@
+# Validates whether or not the passed argument is a valid port
+# (i.e. between `1` - `65535`).
+Puppet::Functions.create_function(:'simplib::validate_port') do
```

Comment:
@jeefberkey There are some modules in which the port is passed in as part of a configuration hash or the port is passed in as a modifier to a string (e.g., '1.2.3.4:80') So, this method is still needed.